src/schemas/schemas.ts

```diff
@@ -4,14 +4,42 @@ import { StatusCodes } from 'http-status-codes';
 // local
 import { MIN_ACTIONS_SAMPLE_SIZE, MAX_ACTIONS_SAMPLE_SIZE } from '../constants/constants';
 
+// todo: get from graasp-utils
 export const uuid = S.string().pattern(
   '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$',
 );
 
 export const id = S.object().prop('id', uuid).required(['id']);
 
+const item = S.object()
```

Comment:
Note: we should probably decide on using either fluent or JSON schema. We may also want to put the reusable schemas in the SDK, or do it dynamically through `getSchemas` (https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/#retrieving-the-shared-schemas). We'll need to discuss this before refactoring.